MAUI/Circular-Charts/DataLabels.md

```diff
@@ -141,9 +141,87 @@ this.Content = chart;
 
 ![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
 
+## Connector line style
+
+This feature is used to customize the connector line. The following [ConnectorLineStyle]() property are used to customize the connector line.
```

Comment:
@EzhilarasanElangovan31  In MAUI, we don't mention as feature in the content. We will be using the property and explain its usage. Check MAUI UG and modify.